modules/ding_config_import/ding_config_import.module

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/**
+ * @file
+ * Implementation of simple form to upload YAML file.
+ */
+
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Implements hook_menu().
+ */
+function ding_config_import_menu() {
+  $item = [];
+
+  $item['admin/config/ding/config'] = [
+    'title' => 'Upload configuration',
+    'description' => 'Upload and import configuration from YAML.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ding_config_import_admin_form'),
+    'access arguments' => array('administer site configuration'),
+  ];
+
+  return $item;
+}
+
+/**
+ * Administration form for yaml file upload.
+ */
+function ding_config_import_admin_form() {
+  $url = drupal_get_path('module', 'ding_config_import') . '/example.config.yml';
+
+  $form = [
+    '#attributes' => [
+      'enctype' => 'multipart/form-data',
+    ],
```

Comment:
The documentation says this is not needed as [Drupal Form API will add it automatically](https://api.drupal.org/api/drupal/developer%21topics%21forms_api_reference.html/7.x#file).

Reply:
Check :-)